src/KafkaSchemaRegistryApiClient.php

```diff
@@ -128,6 +128,10 @@ public function checkSchemaCompatibilityForVersion(
                        $this->createRequestBodyFromSchema($schema)
                    );
         } catch (SubjectNotFoundException|VersionNotFoundException $e) {
+            if ($e instanceof VersionNotFoundException && self::VERSION_LATEST !== $version) {
```

Comment:
👍 How about re-throwing the VersionNotFoundException exception in this case?

Reply:
Ah yes, i totally agree :smile:, updated